### DISCUSSION TITLE
integrate into multiple sites?

### DISCUSSION THREAD

**Discussion by redimongo** [logto-io/logto#3588] [Q&A]
So I am just wondering if it is possible, for this to work with multiple domain names but all the accounts that use login.domain.com (where it is hosted) has access to the same account names.

Much like Google does with YouTube.

**Comment by gao-sun** [comment]
that's exact the scenario what Logto is targeting. with proper CORS configuration, you can achieve this in a secure way.

feel free to send us an email or [schedule a meeting](https://calendly.com/logto/30min?month=2023-03) with us if you need further discussions.
